lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -605,6 +605,7 @@ states:
     response:
       type: cri
       criId: claimedIdentity
+      context: hmrc_check
```

Comment:
This is because we are now able to send them this context, but it previously broke in Staging as they weren't able to accept it.
[relevant slack thread](https://gds.slack.com/archives/C0440GZHUUX/p1722434603877129)
[kiwi ticket](https://govukverify.atlassian.net/browse/KIWI-1940)